01_Data/src/index.ts

```diff
@@ -4,4 +4,5 @@
 // SPDX-License-Identifier: Apache 2.0
 
 export * as sequelize from "./layers/sequelize";
-export * from "./interfaces";
\ No newline at end of file
+export * from "./interfaces";
+export {DeviceModelRepository, Boot, Component, Variable, VariableAttribute} from "./layers/sequelize"; // todo export better as these seem to be used in other modules
```

Comment:
I took a look -
```export * as sequelize from "./layers/sequelize";
export * from "./interfaces";
export {DeviceModelRepository, Boot, Component, Variable, VariableAttribute} from "./layers/sequelize";```
the first line `exports * as sequilize` so in other places you can then import sequelize from here, but I found that in other parts of the code were were trying to import these other things and the way we did it was like
`import {DeviceModelRepository} from '@citrineos/data/layers/sequilize`;`
and it somehow worked but in either case this is here so that these other classes that are being used elsewhere are available. I made this change as well as others to get builds to work properly =/

Reply:
this change should be in its own pr. Exporting only those specific classes circumvents the problem rather than solving it. At this point I think it is clear we should move away from the 'sequelize' export and individually export the classes in 02_Data/layers/sequelize.